include/swift/SILOptimizer/PassManager/PassPipeline.h

```diff
@@ -56,7 +56,9 @@ class SILPassPipelinePlan final {
   SILPassPipelinePlan() = default;
   ~SILPassPipelinePlan() = default;
   SILPassPipelinePlan(const SILPassPipelinePlan &) = default;
+#if !defined(_MSC_VER) || defined(__clang__)
   SILPassPipelinePlan(SILPassPipelinePlan &&) = delete;
```

Comment:
If MSVC doesn't like the `= delete`, can we at least leave an `llvm_unreachable` implementation behind? It must have been deleted for a reason, and it'd be nice to still check dynamically.